README.md

```diff
@@ -59,8 +58,16 @@ or install top level dataset by omitting `-r datasets` option and then calling
 
 You can navigate datasets you have installed in your terminal or browser,
 while fetching necessary files or installing new sub-datasets using the
-same `datalad install [FILENAMES]` command.
-
+`datalad get [FILE|DIR]` command.  DataLad will take care about
```

Comment:
Re 1: A matter of desired API. We have `path` and `source` and simplifications (single pos. argument `path` might be interpreted as `source` and target derived from it). Now, what is it supposed to mean to install a, b, c ? Do we also want a list of sources that are associated by order? Does it mean you have to list paths, that need no source (known submodule) last because otherwise they would be associated with a source? I'm not saying it's not doable, but it is not as easy as it seems at first and I'm not sure whether the API will feel somehow "logical".